provers/blevm/script/src/bin/main.rs

```diff
@@ -77,7 +77,10 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let aggregator_config = AggregatorConfig {
         elf_bytes: BLEVM_AGGREGATOR_ELF,
     };
-    let prover = BlockProver::new(celestia_client, prover_config, aggregator_config);
+    let sp1_client = ProverClient::from_env();
+    let (_, aggregator_vkey) = sp1_client.setup(BLEVM_AGGREGATOR_ELF);
```

Comment:
[blocking] question to @tuxcanfly why does the sp1_client in this file use `BLEVM_AGGREGATOR_ELF` and not `BLEVM_ELF`? I would expect it to use `BLEVM_ELF` because I think this is a single block script

Reply:
@yarikbratashchuk can you research this question? I still don't understand why the single block `blevm` script is using `BLEVM_AGGREGATOR_ELF` here. This seems like a mistake to me. Concretely, I would've expected

```suggestion
    let (_, aggregator_vkey) = sp1_client.setup(BLEVM_ELF);
```

Reply:
this is not needed at all, removed it completely.